pkg/model/resource_usage_parser.go

```diff
@@ -55,11 +75,7 @@ func convertBytesStringWithError(val string) (uint64, error) {
 	if val == "" {
 		return 0, nil
 	}
-	mem, err := datasize.ParseString(sanitizeBytesString(val))
-	if err != nil {
-		return 0, err
-	}
-	return mem.Bytes(), nil
+	return humanize.ParseBytes(val)
```

Comment:
https://github.com/dustin/go-humanize can round trip when parsing (string -> bytes and bytes -> string) and works with SI and IEC units natively. As the name suggests it provides a more human readable value when expressed as a string.